Directory.Packages.props

```diff
@@ -81,4 +81,4 @@
     <PackageVersion Include="Microsoft.VariantAssignment.Client" Version="2.4.17140001" />
     <PackageVersion Include="Microsoft.VariantAssignment.Contract" Version="3.0.16990001" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
Not a big deal - For some reason the newline is removed in the pr as well as in your other, latest pr.